pkg/build/builder/cmd/builder.go

```diff
@@ -140,7 +140,14 @@ func newBuilderConfigFromEnvironment(out io.Writer, needsDocker bool) (*builderC
 			cfg.blobCache = blobCacheDir
 		}
 
-		dockerClient, err := bld.GetDaemonlessClient(systemContext, store, os.Getenv("BUILD_ISOLATION"), cfg.blobCache)
+		imageOptimizationPolicy := buildapiv1.ImageOptimizationNone
+		if s := cfg.build.Spec.Strategy.DockerStrategy; s != nil {
+			if policy := s.ImageOptimizationPolicy; policy != nil {
+				imageOptimizationPolicy = *policy
+			}
+		}
```

Comment:
we should also set the optimization policy to "SkipLayers" when doing an s2i build.  So any time cfg.build.Spec.Strategy.SourceStrategy!=nil we should set imageOptimizationPolicy to SkipLayers.

Reply:
Okay, that should be there now.  Thanks!